paloma-testnet-11/genesis.json

```diff
@@ -8217,23 +8217,23 @@
             "@type": "/cosmos.upgrade.v1beta1.SoftwareUpgradeProposal",
             "description": "The Paloma flock of validators need to upgrade Paloma-Testnet-10 with a breaking change. This upgrade delivers fixes for validator apphash failures due to the inability to verify cosmwasm contracts in state sysnc. Paloma wasmd v0.25.0 lacked app.go support for snapshot of cosmwasm contracts for use with state-sync. This upgrade will allow for more permissionless cosmwasm and NFT activity on Paloma chain that will not cause a majority of validators to fail out of the network. In addition, this release provides the first iteration of the protofiles that will be used to allow anyone to schedule messaging jobs on Paloma for the pigeons to relay.",
             "plan": {
-              "height": "562902",
-              "info": "{\"binaries\":{\"linux/amd64\":\"https://github.com/palomachain/paloma/releases/download/v0.10.1/paloma_Linux_x86_64.tar.gz\"}}",
-              "name": "v0.10.1",
+              "height": "532422",
+              "info": "{\"binaries\":{\"linux/amd64\":\"https://github.com/palomachain/paloma/releases/download/v0.10.0/paloma_Linux_x86_64.tar.gz\"}}",
+              "name": "v0.10.0",
               "time": "0001-01-01T00:00:00Z",
               "upgraded_client_state": null
             },
-            "title": "Paloma breaking-change upgrade to Paloma Release 0.10.1 and Pigeon Release 0.9.0 on the current Testnest-10"
+            "title": "Paloma breaking-change upgrade to Paloma Release 0.10.0 and Pigeon Release 0.9.0 on the current Testnest-10"
           },
           "deposit_end_time": "2022-10-02T07:11:29.058568150Z",
           "final_tally_result": {
             "abstain": "0",
             "no": "0",
             "no_with_veto": "0",
-            "yes": "0"
+            "yes": "22061133762915"
```

Comment:
sum of all votes based on `delegator_share` field in `validator` section of genesis, rounded to teh nearest ugrain